docs/dev/include_parser_list.md

```diff
@@ -1,6 +1,7 @@
 | OS Name | Parser Class |
 | ---------- | ------ |
 | arista_eos | netutils.config.parser.EOSConfigParser |
+| aruba_aoscx | netutils.config.parser.ArubaConfigParser |
```

Comment:
Does this map up to one of the netmiko names? I would try and be consistent with that if possible https://github.com/networktocode/netutils/blob/4a26bcb57b4f71f616de258b713df7333b582409/netutils/lib_mapper.py#L14-L16

Reply:
Unfortunately that's for the older Aruba OS switches.  Netmiko currently does not have support for aruba aoscx, however Scrapli does, should I add this to the scrapli lib mapper?

Reply:
yea, that would be good